exchanges/bitmex/bitmex_websocket.go

```diff
@@ -24,7 +27,7 @@ import (
 )
 
 const (
-	bitmexWSURL = "wss://www.bitmex.com/realtime"
+	bitmexWSURL = "wss://www.bitmex.com/realtimemd"
```

Comment:
I think the only thing that this change highlights is that we're not using the URL system for `b.Websocket.SetupNewConnection` where we are using `bitmexWSURL` instead of `wsEndpoint`.
I bring this up because I received this warning from having an outdated URL in my config:
```
[WARN]  | EXCHANGE | 03/10/2024 11:30:20 | Bitmex: Config is overwriting default endpoint URL values from: [wss://www.bitmex.com/realtimemd] to: [wss://www.bitmex.com/realtime] for: [WebsocketSpotURL]
```
but then we don't do anything with it anyway. 

We don't use the functionality consistently anyway. The feature is there if an endpoint goes down we can use a different one without code-change.
So I will highlight it, see if anyone cares enough to make you do it now, or its something for the pile 🌈

Reply:
It's on my pile.
As I see it, this happens a lot, both in this context of urls and others, where the default value changes, but the user's left with the old default in local config.
One of the things we don't do well right now is to separate out default config from user config in a way that we don't break the user's config when we change defaults, since we flatten them down to the user's file indiscriminately.
I have a plan for this.